db/tidb/bank.go

```diff
@@ -153,7 +153,7 @@ func (c *bankClient) Invoke(ctx context.Context, node cluster.ClientNode, r inte
 	}
 
 	if err = txn.Commit(); err != nil {
-		return bankResponse{Unknown: true, Tso: tso, FromBalance: fromBalance, ToBalance: toBalance}
+		return bankResponse{Unknown: true, Tso: tso, FromBalance: fromBalance, ToBalance: toBalance, Error: err.Error()}
```

Comment:
does Error means unexpected error like io error?

Reply:
Yeah, mostly timeout error because of some fault injections, eg:

* Error 9005: Region is unavailable
* Error 9002: TiKV server timeout